src/components/v2v3/shared/SplitItem.tsx

```diff
@@ -143,13 +169,30 @@ const SplitValue = ({
         {splitValueFormatted}
         {valueSuffix ? <span> {valueSuffix}</span> : null})
       </span>
+      <div
+        style={{
+          fontSize: '.8rem',
```

Comment:
**nitpick** very very nitpicky but eventually want to use either 0.75rem or 0.875rem only (eventually we should get these in variables). Chose whatever works best 👍